src/core/world_state.h

```diff
@@ -0,0 +1,68 @@
+#ifndef CORE_WORLD_STATE_H
+#define CORE_WORLD_STATE_H
+
+#include "utils/common_utils.h"
+
+typedef enum{
+	ELEMENT_ACTIVE,
+	ELEMENT_IN_ROBOT,
+	ELEMENT_DONE,
+	ELEMENT_LOST
```

Comment:
Why are you asking me stuff that we discussed in G3 ? I wrote what everybody sad...